src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs

```diff
@@ -69,5 +69,28 @@ public ProjectRestorer(IReporter reporter = null)
                 throw new ToolPackageException(LocalizableStrings.ToolInstallationRestoreFailed);
             }
         }
+
+        private static void WriteLine(IReporter reporter, string line, FilePath project)
+        {
+            line = line ?? "";
+
+            // Remove the temp project prefix if present
+            if (line.StartsWith($"{project.Value} : ", StringComparison.CurrentCultureIgnoreCase))
```

Comment:
Why CurrentCulture?

Reply:
I wanted to guard against MSBuild outputting a different case for the project file path (I don't think it will given how we construct the path, but for case-insensitive file systems it wouldn't be "wrong" to do so).

However, I don't think an ordinal or invariant comparison would give the desired result.

Reply:
The correct way to compare paths like the Windows file system is OrdinalIgnoreCase.

Reply:
And is that also true of case-insensitive APFS?

Reply:
I don't know.